README.md

```diff
@@ -76,15 +76,15 @@ $ git clone --recurse-submodules https://github.com/rapidsai/rmm.git
 $ cd rmm
 ```
 
-Follow the instructions under "Create the conda development environment `cudf_dev`" in the
+Follow the instructions under "Create the conda development environment `rmm_dev`" in the
```

Comment:
The link says "cudf_dev", so it would be confusing to change this to `rmm_dev`.

```suggestion
Follow the instructions under "Create the conda development environment `cudf_dev`" in the
```